--- dist/qt-everywhere-opensource-src-5.6.0/ubuntu-sdk/src/Ubuntu/Components/plugin/ucbottomedge.cpp
+++ dist/qt-everywhere-opensource-src-5.6.0/ubuntu-sdk/src/Ubuntu/Components/plugin/ucbottomedge.cpp
@@ -260,7 +260,7 @@
 
     UCBottomEdge::DragDirection newDirection = dragDirection;
     qreal delta = previousDistance - currentDistance;
-    bool deltaPassed = abs(delta) >= qApp->styleHints()->startDragDistance();
+    bool deltaPassed = fabs(delta) >= qApp->styleHints()->startDragDistance();
     if (!deltaPassed) {
         return;
     }
@@ -825,7 +825,7 @@
         if (d->mousePressed) {
             QMouseEvent *mouse = static_cast<QMouseEvent*>(event);
             qreal mouseItemY = mapFromScene(mouse->windowPos()).y();
-            qreal distance = abs(height() - mouseItemY);
+            qreal distance = fabs(height() - mouseItemY);
             d->updateProgressionStates(distance);
         }
         break;
