diff --git a/PKGBUILD b/PKGBUILD_new
index 9e308ac..c8deca4 100644
--- a/PKGBUILD
+++ b/PKGBUILD_new
@@ -1,7 +1,7 @@
 # Maintainer: Luca Weiss <WEI16416@spengergasse.at>
 
 pkgname=f3-git
-pkgver=6.0.r16.gd00b9f2
+pkgver=6.0.r17.gf3d067b
 pkgrel=1
 pkgdesc="Utilities to detect and repair counterfeit flash storage, i.e. thumb drives and memory cards with less flash than advertised"
 arch=('i686' 'x86_64')
@@ -34,6 +34,7 @@ build() {
 package() {
   cd "$srcdir/f3"
   make PREFIX="$pkgdir/usr/" install
+  make PREFIX="$pkgdir/usr/" install-extra
 }
 
 # vim:set ts=8 sts=2 sw=2 et:
